src/go/rdctl/cmd/snapshotRestore.go

```diff
@@ -21,15 +21,21 @@ var snapshotRestoreCmd = &cobra.Command{
 
 func init() {
 	snapshotCmd.AddCommand(snapshotRestoreCmd)
+	snapshotRestoreCmd.Flags().BoolVarP(&outputJsonFormat, "json", "", false, "output json format")
+
 }
 
 func restoreSnapshot(_ *cobra.Command, args []string) error {
-	paths, err := p.GetPaths()
+	appPaths, err := paths.GetPaths()
 	if err != nil {
 		return fmt.Errorf("failed to get paths: %w", err)
 	}
-	manager := snapshot.NewManager(paths)
-	if err := manager.Restore(args[0]); err != nil {
+	manager := snapshot.NewManager(appPaths)
+	id, err := getSnapshotId(manager, args[0])
+	if err != nil {
+		return fmt.Errorf("can't restore snapshot: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("can't locate snapshot: %w", err)
```

Reply:
The ID is an implementation detail that will not help the user to understand the error.
I don't want to talk about IDs in messages aimed at the user (as opposed to log messages which are aimed at Rancher Desktop developers).

Reply:
And since we aren't talking about IDs for users, I'm just going to return the error message from `GetSnapshotId`